cogs/highlight.py

```diff
@@ -258,6 +258,7 @@ def _is_registered(self, guild_idx, guild_id, user_id):
         msgs = []
         async for msg in self.bot.logs_from(message.channel,limit=6,around=message):
             msgs.append(msg)
+        msg.replace("`", "\`")

```

Comment:
There is no replace method on a discord.Message object, you'll want to replace text in the message text level below, somewhere near line 272